jormungandr/src/explorer/graphql/mod.rs

```diff
@@ -320,18 +373,38 @@ impl Address {
         Err(ErrorKind::Unimplemented.into())
     }
 
-    fn transactions(&self, context: &Context) -> FieldResult<Vec<Transaction>> {
-        let ids = context
+    fn transactions(
+        &self,
+        first: Option<i32>,
+        last: Option<i32>,
+        before: Option<IndexCursor>,
+        after: Option<IndexCursor>,
```

Comment:
Here, again, using a struct would save some typing and guessing which parameter in the function call refers to what.

Reply:
I see. I guess we'll have to live with it.